corehq/apps/cloudcare/templates/cloudcare/cloudcare_home.html

```diff
@@ -14,39 +14,53 @@
-    <script src="{% static 'cloudcare/js/backbone/cases.js' %}"></script>
-    <script src="{% static 'cloudcare/js/post.js' %}"></script>
-
-    {# touchforms includes #}
-    <script src="{% static 'hqwebapp/js/lib/jquery-ui/jquery-ui-1.8.16.min.js' %}"></script>
-    <link rel="stylesheet" href="{% static 'hqwebapp/js/lib/jquery-ui/jquery-ui-redmond-1.8.16.css' %}">
-
-    <script src="{% static 'formplayer/script/vendor/markdown-it-4.2.0.min.js' %}"></script>
-    <script src="{% static 'formplayer/script/vendor/purify.js' %}"></script>
-    <script src="{% static 'formplayer/script/vendor/ba-tiny-pubsub.min.js' %}"></script>
-    <script src="{% static 'formplayer/script/vendor/shortcut.js' %}"></script>
-    <script src="{% static 'formplayer/script/xforgasm.js' %}?version={{app_version}}"></script>
-    <script src="{% static 'formplayer/script/formnav-all.js' %}?version={{app_version}}"></script>
-    <script src="{% static 'formplayer/script/entrycontrols_full.js' %}?version={{app_version}}"></script>
-    <script src="{% static 'formplayer/script/fullform-ui.js' %}?version={{app_version}}"></script>
-
-    <link rel="stylesheet" href="{% static 'formplayer/style/webforms.css' %}">
-    <link rel="stylesheet" href="{% static 'cloudcare/css/cloudcare.css' %}">
+{% block stylesheets %}
+    <link rel="stylesheet" href="{% new_static 'jquery-ui/themes/base/jquery-ui.min.css' %}">
+    <link type="text/css"
+          rel="stylesheet"
+          media="all"
+          href="{% new_static 'datatables/media/css/jquery.dataTables.min.css ' %}" />
+    <link rel="stylesheet" href="{% new_static 'formplayer/style/webforms.css' %}">
+    <link rel="stylesheet" href="{% new_static 'cloudcare/css/cloudcare.css' %}">
+{% endblock %}
 
+{% block js %} {{ block.super }}
+    <script src="{% new_static 'hqwebapp/js/lib/jquery.textchange.min.js' %}"></script>
+
+    <script src="{% new_static 'hqwebapp/js/ui-element.js' %}"></script>
+    <script src="{% new_static 'case/js/cheapxml.js' %}"></script>
+    <script src="{% new_static 'case/js/casexml.js' %}"></script>
+    <script src="{% new_static 'backbone/backbone-min.js' %}"></script>
+    <script src="{% new_static 'cloudcare/js/util.js' %}"></script>
+    <script src="{% new_static 'cloudcare/js/backbone/shared.js' %}"></script>
+    <script src="{% new_static 'cloudcare/js/backbone/apps.js' %}"></script>
+    <script src="{% new_static 'cloudcare/js/backbone/cases.js' %}"></script>
+    <script src="{% new_static 'cloudcare/js/post.js' %}"></script>
+    <script src="{% new_static 'jquery-ui/jquery-ui.min.js' %}"></script>
+
+    <script src="{% new_static 'hqwebapp/js/lib/jquery.textchange.min.js' %}"></script>
+    <script src="{% new_static 'hqwebapp/js/ui-element.js' %}"></script>
+    <script src="{% new_static 'case/js/cheapxml.js' %}"></script>
+    <script src="{% new_static 'case/js/casexml.js' %}"></script>
+    <script src="{% new_static 'hqwebapp/js/lib/backbone.js' %}"></script>
```

Comment:
huh this seems to be a merge error i specifically put this in the bower folder (see few lines up). will remove